evm/db/state.py

```diff
@@ -217,3 +218,86 @@ def _get_account(self, address):
 
     def _set_account(self, address, account):
         self._trie[address] = rlp.encode(account, sedes=Account)
+
+
+class StateDB(BaseDB):
```

Comment:
Can I suggest we move this to it's own module since it's a low level DB wrapper.

I would also suggest we rename this to something more descriptive.  Here are some names that come to mind.

- `StenographerDB` (A stenographer is the person in a courtroom who types everything that people say).
- `TrackedDB` - tracks changes, touches, etc.
- `LedgerDB` - keeps a ledger of the db read/write

Reply:
Sure!
In `pyethereum`, a similar db wrapper is called `ListeningDB`. IMO `StenographerDB` is cool but a bit too long. I like `TrackedDB`.